stable/kolide-fleet/MAINTAINERS.md

```diff
@@ -0,0 +1,4 @@
+The following document lists the maintainers of the project, as well as the username for each.
```

Comment:
Remove this file. Maintainers are listed in Chart.yaml. Plus, you may add an OWNERS file.